### PR TITLE
Adding additional info related to layout body attributes

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) provides additional layout instruction, on how body attributes can be added.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html

whatsnew
Added the 'Set body attributes' section to [Common layout customization tasks](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -215,6 +216,43 @@ Example: add a block with a product [SKU](https://glossary.magento.com/sku) info
 </block>
 ```
 
+## Set body attributes {#layout_body_attributes}
+
+To set body attributes, use the `<attribute>` instruction.
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md]: Please, add a minor clarification here: 
```suggestion
To set attributes for HTML `body` tag use the `<attribute>` instruction.
```

**Comment by eduard13** [comment]
Hi @rogyar, the suggested changes were applied.
Thank you.

**Comment by dobooth** [comment]
running tests

**Pull request merged**